app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceCard.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.keniphhomemaintenance.maintenance_screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.keniphhomemaintenance.Dimens.MAINTENANCE_CARD_ROW_START_PADDING
+import com.example.keniphhomemaintenance.Dimens.MAINTENANCE_CARD_ROW_VERTICAL_PADDING
+import com.example.keniphhomemaintenance.Dimens.MAINTENANCE_CARD_START_PADDING
+import com.example.keniphhomemaintenance.Dimens.MAINTENANCE_IMAGE_END_PADDING
+import com.example.keniphhomemaintenance.Dimens.MAINTENANCE_IMAGE_SIZE
+import com.example.keniphhomemaintenance.R
+
+@Composable
+fun MaintenanceCard(maintenanceItem: MaintenanceItem) {
+    Spacer(modifier = Modifier.width(MAINTENANCE_CARD_START_PADDING.dp))
+    Row(
+        modifier = Modifier
+            .padding(vertical = MAINTENANCE_CARD_ROW_VERTICAL_PADDING.dp)
+            .padding(start = MAINTENANCE_CARD_ROW_START_PADDING.dp)
+            .fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        val isLightTheme = MaterialTheme.colors.isLight
+        Image(
+            painter = painterResource(
+                id =
+                if (isLightTheme) {
```

Comment:
For fun, you can do a bit of research on Extension functions and create a nifty one to handle this bit of boilerplate, from line 31 to line 40